stable/atlantis/templates/statefulset.yaml

```diff
@@ -0,0 +1,192 @@
+          - name: ATLANTIS_BITBUCKET_TOKEN
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "atlantis.name" . }}-webhook
+                key: bitbucket_token
+          {{- if .Values.bitbucket.base_url }}
+          - name: ATLANTIS_BITBUCKET_BASE_URL
+            value: {{ .Values.bitbucket.base_url }}
+          - name: ATLANTIS_BITBUCKET_WEBHOOK_SECRET
+            valueFrom:
+              secretKeyRef:
+                name: {{ template "atlantis.name" . }}-webhook
+                key: bitbucket_secret
+          {{- end }}
+          {{- end }}
+          {{- if .Values.livenessProbe.enabled }}
+          livenessProbe:
+            httpGet:
+              path: /healthz
+              port: 4141
+              scheme: {{ .Values.livenessProbe.scheme }}
+            initialDelaySeconds: {{ .Values.livenessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.livenessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.livenessProbe.successThreshold }}
+            failureThreshold: {{ .Values.livenessProbe.failureThreshold }}
+          {{- end }}
+          {{- if .Values.readinessProbe.enabled }}
+          readinessProbe:
+            httpGet:
+              path: /healthz
+              port: 4141
+              scheme: {{ .Values.readinessProbe.scheme }}
+            initialDelaySeconds: {{ .Values.readinessProbe.initialDelaySeconds }}
+            periodSeconds: {{ .Values.readinessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.readinessProbe.timeoutSeconds }}
+            successThreshold: {{ .Values.readinessProbe.successThreshold }}
+            failureThreshold: {{ .Values.readinessProbe.failureThreshold }}
+          {{- end }}
+          volumeMounts:
+          - name: atlantis-data
+            mountPath: /atlantis-data
+          {{- range $name, $_ := .Values.serviceAccountSecrets }}
+          - name: {{ $name }}-volume
+            readOnly: true
+            mountPath: /etc/{{ $name }}
+          {{- end }}
+          {{- if .Values.gitconfig}}
+          - name: gitconfig-volume
+            readonly: true
+            mountPath: /etc/secret-gitconfig
+          {{- end }}
+          {{- if .Values.aws}}
+          - name: aws-volume
+            readonly: true
+            mountPath: /home/atlantis/.aws
+          {{- end }}
+          resources:
+{{ toYaml .Values.resources | indent 12 }}
+    {{- with .Values.nodeSelector }}
+      nodeSelector:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+    {{- with .Values.affinity }}
+      affinity:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+    {{- with .Values.tolerations }}
+      tolerations:
+{{ toYaml . | indent 8 }}
+    {{- end }}
+  volumeClaimTemplates:
+  - metadata:
+      name: atlantis-data
+    spec:
+      accessModes: ["ReadWriteOnce"] # Volume should not be shared by multiple nodes.
```

Comment:
Can you also add storageClass like https://github.com/helm/charts/blob/9897667e80066a3ec0e4a0c98d2c45400784e9d9/stable/consul/templates/consul-statefulset.yaml#L190-L193 ?

Makes using another class easier.